jupyter_conda/envmanager.py

```diff
@@ -524,6 +524,29 @@ async def env_packages(self, env: str) -> Dict[str, List[str]]:
 
         return {"packages": [normalize_pkg_info(package) for package in data]}
 
+    async def pkg_depends(self, pkg: str) -> Dict[str, List[str]]:
+        """List environment packages dependencies.
+
+        Args:
+            pkg (str): Package name
+
+        Returns:
+            {"package": List[dependencies]}
+        """
+        resp = {}
+        ans = await self._execute(self.manager, "repoquery", "depends", "--json", pkg)
+        _, output = ans
+        query = self._clean_conda_json(output)
+        
+        if "error" not in query:
+            for dep in query['result']['pkgs'] :
+                if type(dep) is dict :
+                    deps = dep.get('depends', None)
+                    if deps :
+                        resp[dep['name']] = deps
```

Comment:
I would returned an empty list rather than skipping the entry.

It seems strange to have:
```
pkg_depends("a") -> {"a": ["sub_a1", "sub_a2"]}
```
if `a` has two dependencies. But
```
pkg_depends("b") -> {}
```
If `b` has no dependency.